airflow/www/views.py

```diff
@@ -356,11 +356,18 @@ def get_int_arg(value, default=0):
         state_color_mapping = State.state_color.copy()
         state_color_mapping["null"] = state_color_mapping.pop(None)
 
+        page_title = "Airflow - DAGs"
+        site_title = conf.get(section="webserver", key="site_title", fallback="DAGs")
+        if site_title != "DAGs":
+            page_title = site_title
+
         return self.render_template(
             'airflow/dags.html',
             dags=dags,
             current_page=current_page,
             search_query=arg_search_query if arg_search_query else '',
+            page_title=page_title,
```

Comment:
Why do we need both?

Reply:
As above: to have a way to distinguide an installation from the main DAGs page title.

Reply:
I'll double check this, but if not needed for the UI will remove.

Reply:
Have amended the code slightly so that `Airflow - DAGs - CustomText` appears as the site_title and the `<h2>` renders the page_title.